CDCgov/prime-reportstream
# 404 and Unsupported Browser error pages

This PR creates an `ErrorPage` component framework that allows for us to expand our error page content in the future! It comes with a 404 and Unsupported Browser page.

Test Steps:
1. pull `kevinhaube/error-pages`
2. `npm install && npm start`
3. Navigate to any undefined path such as `http://localhost:3000/kevin-thank-you-for-the-cool-error-page`
4. See that?? If not, I messed up.

## Changes
- Added `<ErrorPage>` component to handle these and future needs
- Added `NotFound` and `UnsupportedBrowswer` content to populate error pages

## Checklist

### Testing

### Process

## Fixes
- #2436 

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
No
- Does this PR include changes in authentication and/or authorization of existing endpoints?
No
- Does this change introduce new dependencies that need vetting?
No
- Does this change require changes to our infrastructure?
No
- Does logging contain sensitive data?
No
- Does this PR include or remove any sensitive information itself?
No

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
